cdap-docs/reference-manual/source/http-restful-api/security.rst

```diff
@@ -28,18 +29,55 @@ The HTTP RESTful API is divided into these sections:
 Authorization
 =============
 
-Use the CDAP Authorization HTTP RESTful API to grant, revoke and list privileges on CDAP entities. Details about
-authorization in CDAP can be found at :ref:`Admin' Manual: Authorization <admin-authorization>`.
+Use the CDAP Authorization HTTP RESTful API to grant, revoke, and list privileges on CDAP
```

Comment:
what's the maximum number of characters on a single line in the rst files? I don't go beyond 120, but it seems like we're cutting it even further here.